app/helpers/pagelets_helper.rb

```diff
@@ -3,7 +3,7 @@ def pagelets_for(mountpoint)
     Pagelets::Manager.sorted_pagelets_at("#{controller_name}/#{action_name}", mountpoint)
   end
 
-  def render_pagelets_for(mountpoint, opts = {})
+  def render_pagelets_for(mountpoint, model, opts = {})
     result = ""
     pagelets_for(mountpoint).each do |pagelet|
       result += render_pagelet(pagelet, opts)
```

Comment:
`model` should be passed down to `render_pagelet` here. Otherwise it's not accessible in the `:onlyif` proc.